package/MDAnalysis/coordinates/DCD.py

```diff
@@ -472,7 +478,7 @@ def _read_next_timestep(self, ts=None):
         """
         if ts is None:
             ts = self.ts
-        ts._frame = self._read_next_frame(ts._x, ts._y, ts._z, ts._unitcell, self.skip)
+        ts._frame = self._read_next_frame(ts._x, ts._y, ts._z, ts._unitcell, 1)
```

Comment:
From what I can see... you could load a reader with a `skip` != 1 and then your natural iteration would always have this skip built in.  So I probably have to document this change, because it's gone now.